src/Context/AuthContext.js

```diff
@@ -1,16 +1,16 @@
-import { createContext, useContext, useState, useReducer, useEffect } from "react";
+import { createContext, useContext, useState } from "react";
 import axios from "axios";
-import { Toast } from "../components"
 import { useNavigate } from "react-router-dom";
 
-
 const authContext = createContext(null);
 
 const AuthProvider = ({children}) => {
-    const localStorageToken = JSON.parse(localStorage.getItem("token"));
-    const [token, setToken] = useState(localStorageToken?.token);
+    const localStorageToken = JSON.parse(localStorage.getItem("signup"));
+    const [token, setToken] = useState(localStorageToken);
     const localStorageUser = JSON.parse(localStorage.getItem("user"));
-    const [user, setUser] = useState(localStorageUser?.user);
+    const [user, setUser] = useState(localStorageUser);
+    const [isLogin, setIsLogin] = useState(JSON.parse(localStorage.getItem("login")) ? true : false)
```

Comment:
don't directly read data from local storage instead use `useEffect`, inside that read from localstorage and set the state